src/Mod/OpenSCAD/OpenSCADCommands.py

```diff
@@ -327,7 +327,6 @@ def retranslateUi(self, widget=None):
 
 class OpenSCADMeshBooleanTask:
     def __init__(self):
-        pass
```

Comment:
@KeithSloan care to review this?

Reply:
in any case, indeed these pass statements can all be removed.

Reply:
You can't remove the `pass` here. As the documentation says, `pass` is used to fill a block that otherwise is empty. In this case the `pass` keyword does nothing, it is just an indicator that "something could be placed here". So, if you remove the `pass`, `__init__` is empty, and it will cause an error.

Since the `__init__` method is empty, you could remove it entirely, together with the `pass`.

Reply:
I just saw the full code. I thought `__init__` was empty, but it is not. So, indeed, `pass` can be removed without problem here as well.